pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -560,4 +561,26 @@ public static String decodeUrl(String input)
       throws UnsupportedEncodingException {
     return URLDecoder.decode(input, StandardCharsets.UTF_8.toString());
   }
+
+  /**
+   *
```

Comment:
minor: I would get rid of the extra blank line in the comment to save vertical space on the screen.

Reply:
Done